av1an-core/src/settings.rs

```diff
@@ -578,7 +580,20 @@ properly into a mkv file. Specify mkvmerge as the concatenation method by settin
       Input::VapourSynth(vs_script) => self.create_video_queue_vs(splits, vs_script.as_path()),
     };
 
-    chunks.sort_unstable_by_key(|chunk| Reverse(chunk.frames));
+    match self.chunk_order {
+      ChunkOrdering::LongestFirst => {
+        chunks.sort_unstable_by_key(|chunk| Reverse(chunk.frames));
+      }
+      ChunkOrdering::ShortestFirst => {
+        chunks.sort_unstable_by_key(|chunk| chunk.frames);
+      }
+      ChunkOrdering::Sequential => {
+        // Already in order
+      }
+      ChunkOrdering::Random => {
+        chunks.sort_unstable_by_key(|_| rand::random::<u16>());
```

Comment:
Let's use [shuffle](https://docs.rs/rand/latest/rand/seq/trait.SliceRandom.html#tymethod.shuffle) for this, just to guarantee an O(n) time complexity

Reply:
Thanks, I was looking for this method and couldn't find it. 👍